Marsiple/Common/Models/Todo.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-struct Todo {
+class Todo {
```

Comment:
This change does have consequences, why the change?

Reply:
I've done that so changes made in `TodoFormViewController` reflect on the object that `viewController` was `initialized` with.

Reply:
This is risky behaviour. Isn't it more safe to work with two completely isolated instances and then just ovewrite the legacy one with the new one, rather than modifying the same instance from two different places?

I would argue that, even though it might seem like a good idea to use a reference type here, it isn't - a `immutable` struct's almost always a better idea.